docs/release-notes/2.42.0.md

```diff
@@ -0,0 +1,135 @@
+# Percona Monitoring and Management 2.42.0
+
+| **Release date** | May 30th, 2024                                                                                   |
+| ----------------- | ----------------------------------------------------------------------------------------------- |
+| **Installation** | [Installing Percona Monitoring and Management](../quickstart/index.md) |
+
+Percona Monitoring and Management (PMM) is an open source database monitoring, management, and observability solution for MySQL, PostgreSQL, and MongoDB.
+
+It enables you to observe the health of your database systems, explore new patterns in their behavior, troubleshoot them and execute database management operations regardless of whether your databases are located on-premises or in the cloud.
+
+## What's new in this release
+
+This release introduces support for Ubuntu 24.04, configurable metrics resolutions per service, improved connection management for PostgreSQL and PMM Agent, and experimental dashboards for PMM self-monitoring and MongoDB. 
+
+The release also brings enhancements to MySQL Query Response Time Details, adds new metrics and labels for PostgreSQL, and addresses several issues related to MongoDB monitoring, upgrades, and Query Analytics and High Availability documentation.
+
+Experimental dashboards in PMM are new or redesigned dashboards that are released as a preview feature for you to test and provide feedback.
+
+Keep in mind that these dashboards are not considered fully stable or complete, and their design, functionality, and metrics may change in future releases based on user feedback and further development.
+
+## Release highlights
+
+## Ubuntu 24.04 Noble Numbat support
+
+PMM now officially supports monitoring of databases running on the recently released Ubuntu 24.04 Noble Numbat operating system.
+
+With this addition, PMM continues to provide comprehensive coverage across a wide range of Linux distributions commonly used in database deployments.
+
+Check the full list of supported platforms on the [Percona Software Support Lifecycle webpage](https://www.percona.com/services/policies/percona-software-support-lifecycle).
```

Comment:
Any way to link directly to the Percona Tools heading?  Finding PMM in that sea of text is hard...I missed it twice...also doesn't show 24.04 supported but I assume that's to come with the final deploy

https://www.percona.com/services/policies/percona-software-support-lifecycle#:~:text=older%20minor%20versions..-,Percona%20Tools,-Software%3A%20Percona